spec/requests/api/v1/orders_request_spec.rb

```diff
@@ -0,0 +1,163 @@
+require 'rails_helper'
+
+describe 'Orders API', :type => :request do
+  context 'with a correct API key' do
+    it 'creates an order for a successful request' do
+      distributor = Distributor.create!(name: 'RNDC',
+                                        address: '3319 Arapahoe st, Denver, CO',
+                                        code: 'CODE1234',
+                                        api_key: 'jgn983hy48thw9begh98h4539h4',
+                                        password: 'password'
+                                        )
+
+      item_1 = create(:item, distributor: distributor)
+      item_2 = create(:item, distributor: distributor)
+      business = create(:business, api_key: 'jgn983hy48thw9begh98h4539h4', distributor: distributor)
+
+      params = {'api_key': 'jgn983hy48thw9begh98h4539h4',
+                'total_cost': '200',
+                'total_revenue': '40',
+                'items': [{
+                   'id': "#{item_1.id}",
+                   'quantity': '4',
+                   'price': '25'
+                 },
+                 {
+                    'id': "#{item_2.id}",
+                    'quantity': '2',
+                    'price': '50'
+                  }
+               ]
+               }
+
+      expect(Order.all.count).to eq(0)
+
+      post '/api/v1/orders', params: params
+
+      result = JSON.parse(response.body)['data']
+      # THIS TEST WILL FAIL ONCE MERGED INTO MASTER, AS A REPRESENTATIVE WILL NEED TO
+      #  BE CREATED IN THE TEST SETUP AND THE BUSINESS WILL NEED TO HAVE A REPRESENTATIVE
+
+      # UNIT IS HARDCODED INTO THE COntroller RIGHT NOW and will need to either be tested for,
```

Comment:
I think you have a good point here, we don't really need the unit column here, I'm going to get rid of it